io/net/uperf_test.py

```diff
@@ -136,6 +167,7 @@ def test(self):
         cmd = "h=%s proto=tcp ./src/uperf -m manual/throughput.xml -a" \
             % self.peer_ip
         result = process.run(cmd, shell=True, ignore_status=True)
+        nping_result = self.nping()
```

Comment:
@vaishnavibhat  what i feel this line should be added after throughput compare, here is no point to keep anyways nping is checking after uperf test is complete.

Reply:
@smruti77 Want to check if nping sees a packet loss during performance tests and base the results on both uperf out put and nping value. So doing a nping check immediately.

Reply:
ok,but can you add this after ```exit_status``` check to make sure command run fine or uperf not failed. sometime we see uperf failed,so that time no use to check nping.

Reply:
@vaishnavibhat Can you take care of this comment please ?